src/components/APIContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import GetTimelineButton from "./GetTimelineButton.jsx";
+import TimelineContainer from "./TimelineContainer.jsx";
+import requestHandler from "../js/requestHandler.js";
+
+class APIContainer extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            request: new XMLHttpRequest(),
```

Comment:
is this still needed?

Reply:
my requestHandler doesn't create the XMLHttpRequest